### PR TITLE
Release 12 h1

### PR DESCRIPTION
**Pull request by deanonishi**
Minor doc changes to 'app_passenger', 'app_php', 'app_tomcat', and 'block_device'.
Only changed ReadMe and metadata.rb files.

### CODE REVIEW

File: cookbooks/block_device/metadata.rb [hunk 1]
```diff
@@ -270,7 +270,7 @@
 
 attribute "block_device/terminate_safety",
   :display_name => "Terminate Safety",
-  :description => "Prevents the accidental running of the block_device::do_teminate_server recipe. This recipe will only run if the input variable is overridden and set to \"off\".",
+  :description => "Prevents the accidental running of the block_device::do_teminate_server recipe. The recipe will only run if the input variable is overwritten and set to \"off\".",
```
Comment (douglaswth) [inline cookbooks/block_device/metadata.rb]: This should still be overridden, not overwritten.

File: cookbooks/block_device/metadata.rb [hunk 2]
```diff
@@ -279,7 +279,7 @@
 
 attribute "block_device/force_safety",
   :display_name => "Force Reset Safety",
-  :description => "Prevents the accidental running of the block_device::do_force_reset recipe. This recipe will only run if the input variable is overridden and set to \"off\".",
+  :description => "Prevents the accidental running of the block_device::do_force_reset recipe. The recipe will only run if the input variable is overwritten and set to \"off\".",
```
Comment (douglaswth) [inline cookbooks/block_device/metadata.rb]: This should still be overridden, not overwritten.

File: cookbooks/block_device/README.rdoc [hunk 3]
```diff
@@ -28,40 +28,42 @@ Please see <tt>metadata.rb</tt> file for the latest dependencies.
...
 
 === Force Reset:
 
-1. WARNING: this detaches all storage related to your lineage and DESTROYs the
-   volumes\! This is to be used if you'd like to force run a restore but you
-   already have storage attached that you don't want. After running the recipe
-   "block_device::do_force_reset" you can then run
-   "block_device::setup_block_device" or "block_device::do_restore".
+1. WARNING: The "block_device::do_force_reset" recipe detaches all storage 
+   related to your lineage and DESTROYS the volumes\! You should only run
```
Comment (douglaswth) [inline cookbooks/block_device/README.rdoc]: Is the backslash actually necessary?

File: cookbooks/block_device/metadata.rb (hunk 2 above)
Comment (deanonishi) [inline cookbooks/block_device/metadata.rb]: Yes, agreed.

File: cookbooks/block_device/README.rdoc (hunk 3 above)
Comment (deanonishi) [inline cookbooks/block_device/README.rdoc]: The backslash was there before my changes. I'm not sure if it's necessary. I vote to remove it if it doesn't break anything.

Comment (douglaswth) [inline cookbooks/block_device/README.rdoc]: It is a README file, so it won't break anything. Do it!

**Pull request merged**